src/models/User.ts

```diff
@@ -116,7 +116,7 @@ export class User extends Model<UserAttributes, UserCreationAttributes> implemen
   @Column(DataType.TEXT)
   slackId!: string | null;
 
-  @AllowNull(false)
+  @AllowNull(true)
```

Comment:
If we changed this here I think we need to change it in the BackUpSQL. This means we need to do either recreate all the DB in staging and prod or change that rule with an SQL query with a little help from @marcelomilera.

I would go for option 2 (the SQL query).